packages/search/src/test/use-fuzzy-search.tsx

```diff
@@ -0,0 +1,69 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import Search, { useFuzzySearch } from '..';
+import type { UseFuzzySearchOptions } from '../use-fuzzy-search';
+import '@testing-library/jest-dom';
+
+const TestComponent = < T, >( props: UseFuzzySearchOptions< T > ) => {
+	const { results, setQuery } = useFuzzySearch( props );
+
+	return (
+		<>
+			<Search onSearch={ setQuery } />
+			{ results.map( ( result ) => JSON.stringify( result ) ) }
+		</>
+	);
+};
+
+describe( 'useFuzzySearch', () => {
+	it( 'allows fuzzy searching a simple string array', async () => {
+		const user = userEvent.setup();
+
+		const data = [ 'ok', 'hello' ];
+
+		render( <TestComponent data={ data } /> );
+
+		const searchbox = screen.getByRole( 'searchbox' );
+		await user.type( searchbox, 'k' );
+
+		expect( screen.queryByText( /ok/ ) ).toBeInTheDocument();
```

Comment:
Nit: When we're querying for something that the user sees, I find `.toBeVisible()` a bit better, because in practice something might be in the document, but still be invisible for the user. At the same time, `.not.toBeInTheDocument()` is just fine.